quickwit-search/src/search_stream/mod.rs

```diff
@@ -0,0 +1,129 @@
+// Quickwit
+//  Copyright (C) 2021 Quickwit Inc.
+//
+//  Quickwit is offered under the AGPL v3.0 and as commercial software.
+//  For commercial licensing, contact us at hello@quickwit.io.
+//
+//  AGPL:
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Affero General Public License as
+//  published by the Free Software Foundation, either version 3 of the
+//  License, or (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Affero General Public License for more details.
+//
+//  You should have received a copy of the GNU Affero General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+mod collector;
+mod leaf;
+mod root;
+
+pub use collector::{FastFieldCollector, FastFieldCollectorBuilder};
+pub use leaf::leaf_search_stream;
+pub use root::root_search_stream;
+
+use serde::Deserialize;
+use std::{
+    fmt::Display,
+    io::{self, Write},
+};
+use tantivy::fastfield::FastValue;
+
+/// Output format for stream.
+#[derive(Deserialize, Debug, PartialEq, Eq, Clone, Copy)]
+#[serde(rename_all = "lowercase")]
+pub enum OutputFormat {
+    /// Format data by row in ClickHouse binary format. TODO: add link to format.
+    CHRowBinary,
```

Comment:
Let's explicitly call it ClickHouseRowBinary. I had trouble understanding what was CH about.

Reply:
arf, I forget to rename it, `ClickHouseRowBinary` is better

Reply:
doing it